config.js

```diff
@@ -167,11 +167,11 @@ var config = {
 
     // Recording
 
-    // Whether to enable recording or not.
-    // enableRecording: false,
+    // Whether to enable file recording or not.
+    // fileRecordingsEnabled: false,
```

Comment:
`fileRecordingsEnabled`? with an `s`?

Reply:
copied and pasted what will be used: fileRecordingsEnabled and liveStreamingEnabled